### PR TITLE
Add free method to MlsClient to match proto

### PR DESCRIPTION
**Pull request by brianathere**
The MlsClient proto added a free method, add it to the implementation so all tests can compile and run.

### CODE REVIEW

File: interop_client/src/main.rs [hunk 1]
```diff
@@ -1124,6 +1124,11 @@ impl MlsClient for MlsClientImpl {
     ) -> Result<Response<ProposalResponse>, Status> {
         todo!()
     }
+    async fn free(&self, _request: Request<FreeRequest>) -> Result<Response<FreeResponse>, Status> {
```
Comment (duesee) [inline interop_client/src/main.rs:1127]:
```suggestion

    async fn free(&self, _request: Request<FreeRequest>) -> Result<Response<FreeResponse>, Status> {
```

Could we add a newline?

**Review by duesee — COMMENTED** [review]
Thanks, @brianathere! Great timing! We just talked about implementing this in https://github.com/openmls/openmls/pull/1348. I would suggest to merge your PR first and rebase https://github.com/openmls/openmls/pull/1348 later. (This is not a review, but merely a first comment.)

**Review by franziskuskiefer — APPROVED** [review]
Thanks! I'll go ahead and merge this.

**Comment by franziskuskiefer** [comment]
_(force merged because the coverage job isn't running for external PRs apparently)_

**Pull request merged**